misk/src/main/kotlin/misk/web/actions/DashboardMetadataAction.kt

```diff
@@ -23,24 +24,27 @@ import javax.inject.Singleton
  */
 
 @Singleton
-class AdminDashboardTabAction @Inject constructor() : WebAction {
-  @Inject @AdminDashboardTab private lateinit var adminDashboardTabs: List<DashboardTab>
+class DashboardMetadataAction @Inject constructor() : WebAction {
+  @Inject private lateinit var allDashboardTabs: List<DashboardTab>
   @Inject lateinit var callerProvider: @JvmSuppressWildcards ActionScoped<MiskCaller?>
 
-  @Get("/api/admindashboardtabs")
+  @Get("/api/dashboard/metadata/{dashboard}")
```

Comment:
Would it make more sense for this to be `/api/dashboard/{dashboard}/metadata`, or even `/api/dashboard/{dashboard}/tabs`?

Reply:
The path continues the pattern of `ServiceMetadataAction` which has the path `/api/service/metadata`, hope that makes sense!

Reply:
That makes sense re: `/metadata` vs. `/tabs`, but I think the rearrangement still stands.

The `/api/service/metadata` is returning metadata for all services, not a specific one. Now that the `DashboardMetadataAction` is returning results for a **specific** dashboard, as opposed to all admin dashboard tabs, IMHO it would make sense for the URI to follow the `parent/child` model.  In this case, the metadata you're requesting is a child of the identified dashboard.

The current structure, `/api/dashboard/metadata/{dashboard}`, is effectively using the dashboard path param as a filter param.  The alternative, `/api/dashboard/{dashboard}/metadata`, communicates that you're accessing child properties of the dashboard identified by the path param.

Reply:
Thanks for expanding your thoughts on this, I think they make sense. I've updated the path accordingly to now be `/api/dashboard/{dashboard}/metadata`.